File: Iniciante/1071.cpp
#include <iostream>
using namespace std;

int main() {
    int x, y;
    int s = 0;
    cin >> x >> y;
    for(int i = min(x, y) + 1; i < max(x, y); ++i)
        if(abs(i)%2 != 0) s += i;
    cout << s << endl;
    return 0;
}

File: Iniciante/C#/1071.cs
using System;
class URI {
    static void Main(string[] args) { 
        int x = int.Parse(Console.ReadLine());
        int y = int.Parse(Console.ReadLine());
        int s = 0;
        for(int i = Math.Min(x, y) + 1; i < Math.Max(x, y); ++i)
            if(Math.Abs(i)%2 != 0) s += i;
        Console.WriteLine(s);
    }   
}